draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3171,56 +3171,28 @@ Procedures for sending media differ for full and lite implementations.
 <section title="Procedures for Full Implementations">
 
 <t>
-Agents always send media using a candidate pair, called the selected
-candidate pair. An agent will send media to the remote candidate in
-the selected pair (setting the destination address and port of the
-packet equal to that remote candidate), and will send it from the
-local candidate of the selected pair. When the local candidate is
+A full implementation MUST NOT send media until it has a Valid list
+that contains a candidate pair for each component of that media
+stream. Once that happens, the agent MAY begin sending media
+packets.
+</t>
+	
+<t>
+An agent will send media to the remote candidate (setting the 
+destination address and port of the packet equal to that remote candidate), 
+and will send it from the local candidate. When the local candidate is
 server or peer reflexive, media is originated from the base. Media
 sent from a relayed candidate is sent from the base through that TURN
```

Comment:
This PR only proposes changes directly related to passive-aggressive nomination.  The text relating to sending media has not been changed.  So it seems to me that your concern is orthogonal and should be handled in a separate issue/PR.